### PR TITLE
Derive some more ToCss stuff again

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/20200)

### CODE REVIEW

File: components/style_derive/to_css.rs [hunk 1]
```diff
@@ -18,7 +18,9 @@ pub fn derive(input: syn::DeriveInput) -> Tokens {
     let s = synstructure::Structure::new(&input);
 
     let match_body = s.each_variant(|variant| {
-        let bindings = variant.bindings();
+        let bindings = variant.bindings().into_iter().filter(|binding| {
+            !cg::parse_field_attrs::<CssFieldAttrs>(&binding.ast()).skip
```
Comment (emilio) [inline components/style_derive/to_css.rs]: Would it make sense to just do `if attrs.skip { continue; }` in the loop? Unless we have a use case for `skip` + `iterable`, I don't think we should re-parse it again.

**Review by emilio — APPROVED** [review]

File: components/style_derive/to_css.rs (hunk 1 above)
Comment (nox) [reply components/style_derive/to_css.rs]: I changed it this way.

**Comment by nox** [comment]
https://treeherder.mozilla.org/#/jobs?repo=try&revision=0354b69f483cc5edc98494c471ef4830e90ded54

**Pull request merged**